velox/vector/arrow/Bridge.cpp

```diff
@@ -315,147 +236,459 @@ const char* exportArrowFormatStr(const TypePtr& type) {
   }
 }
 
-} // namespace
+// A filter representation that can also keep the order.
```

Comment:
ah, just realized that this is a phabricator stack and thus this PR includes all the array/map conversion stuff as well. I'll start reviewing that one first instead.